home/tests/test_assessment_import_export.py

```diff
@@ -0,0 +1,473 @@
+import csv
+import re
+from collections.abc import Callable, Iterable
+from dataclasses import dataclass
+from functools import wraps
+from io import BytesIO, StringIO
+from pathlib import Path
+from queue import Queue
+from typing import Any
+
+import pytest
+from openpyxl import load_workbook
+from pytest_django.fixtures import SettingsWrapper
+from wagtail.models import Locale  # type: ignore
+
+from home.assessment_import_export import import_assessment
+from home.import_assessment_pages import ImportAssessmentException
+from home.models import (
+    HomePage,
+)
+
+from .page_builder import (
+    MBlk,
+    MBody,
+    PageBuilder,
+    WABlk,
+    WABody,
+)
+
+IMP_EXP_DATA_BASE = Path("home/tests/import-export-data")
+
+ExpDict = dict[str, Any]
+ExpPair = tuple[ExpDict, ExpDict]
+ExpDicts = Iterable[ExpDict]
+ExpDictsPair = tuple[ExpDicts, ExpDicts]
+
+
+def filter_both(
+    filter_func: Callable[[ExpDict], ExpDict]
+) -> Callable[[ExpDict, ExpDict], ExpPair]:
+    @wraps(filter_func)
+    def ff(src: ExpDict, dst: ExpDict) -> ExpPair:
+        return filter_func(src), filter_func(dst)
+
+    return ff
+
+
+@filter_both
+def ignore_certain_fields(entry: ExpDict) -> ExpDict:
+    # FIXME: Do we need page.id to be imported? At the moment nothing in the
+    #        import reads that.
+    # FIXME: Implement import/export for doc_link, image_link, media_link.
+    ignored_fields = {
+        "page_id",
+        "doc_link",
+        "image_link",
+        "media_link",
```

Comment:
We don't have any documents, images, or media for assessments, so we don't need these.